Jenkinsfile.cig

```diff
@@ -108,6 +108,15 @@ pipeline {
       }
     }
 
+    stage('Build Documentation') {
+      steps {
+        sh './doc/update_parameters.sh ./build-gcc-fast/aspect || touch ~/FAILED-DOC'
```

Comment:
you can remove the ``|| touch ...``. Then the command will fail immediately (below I do this so that we get the .log file).